gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/02_add_shared_policy_group.yml

```diff
@@ -46,4 +46,41 @@ databaseChangeLog:
                   type: nvarchar(64)
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.

Reply:
🆗 as this not released yet i can change it